### PR TITLE
added option for adding polling time when consuming records using kafka consumer

### PR DESCRIPTION
**Pull request by tirth7777777**
Test Description## What
Added option for adding polling time when records are being consumed from Kafka consumer. Kafka consumer polling time is fixed in source-kafka
## How
added polling_time in spec and added polling time as a variable in KafkaSource file

### CODE REVIEW

File: airbyte-integrations/connectors/source-kafka/src/main/resources/spec.json [hunk 1]
```diff
@@ -78,6 +78,12 @@
         "type": "integer",
         "default": 500
       },
+      "polling_time": {
+        "title": "Polling Time",
+        "description": "Amount of time Kafka connector should try to poll for messages.",
+        "type": "integer",
+        "default": 5000
+      },
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-kafka/src/main/resources/spec.json:86]: @misteryeo can you check this change in the specification of Kafka connector. This is something we want to expose to users?

File: airbyte-integrations/connectors/source-amazon-ads/Dockerfile [hunk 2]
```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.5
+LABEL io.airbyte.version=0.1.6
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-amazon-ads/Dockerfile:15]: @tirth7777777 please check your commit because is changing Amazon Ads too.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-amazon-ads/Dockerfile (hunk 2 above)
Comment (tirth7777777) [reply airbyte-integrations/connectors/source-amazon-ads/Dockerfile:15]: oh sorry. I will correct that ASAP

**Comment by marcosmarxm** [comment]
@tirth7777777 did you have to correct my comment? Also can you solve conflicts?

**Comment by misteryeo** [comment]
@tirth7777777 can you elaborate on the use case for exposing this to users? Wanted to make sure I understood that before approving this. And how did we determine what the default value for polling time should be?

On a separate note, please make sure we're also updating the documentation to reflect this new optional field.

**Comment by marcosmarxm** [comment]
I'm closing due inactivity, @tirth7777777 let us know if you want to continue working on this.

**Pull request closed without merging**